src/main/resources/application.properties

```diff
@@ -1,5 +1,7 @@
 # See application-oauth2.properties in project root for example OAuth2 configuration.
-
+#TODO
```

Comment:
Instead of TODO, please add a short documentation via comment what these lines can be used for.